.github/workflows/helm_release.yaml

```diff
@@ -6,13 +6,15 @@ on:
     branches:
       - main
     paths:
-      - 'charts/**'
```

Comment:
I think any file change should trigger a release, including Chart.yaml

Reply:
That means that the version needs to be updataed. I haven't done that because there were issues with helm releaser and versions with `+` in them. https://github.com/helm/chart-releaser-action/issues/193 

As the number of changes in this repo is going to be low, I think to makes sense to create a new release. Even if currently the releases are not getting executed, we can always update the chart to trigger the release.